### PR TITLE
Simple SceneTree dumper to help illustrating SceneTree problems

### PR DESCRIPTION
**Pull request by 84634E1A607A**
**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
Why is this better than running Thrive through the Godot editor and using the Godot editor's remote node view (which allows you to also edit the nodes as you inspect them)?

**Comment by 84634E1A607A** [comment]
I can copy it to pastebin

File: src/engine/DebugOverlays.tscn [hunk 1]
```diff
@@ -81,6 +80,7 @@ custom_fonts/font = ExtResource( 6 )
 text = "METRICS_CONTENT"
 
 [node name="DebugPanel" parent="." instance=ExtResource( 1 )]
+visible = true
```
Comment (hhyyrylainen) [inline src/engine/DebugOverlays.tscn:83]: Accidentally visible?

Comment (84634E1A607A) [reply src/engine/DebugOverlays.tscn:83]: Yeah

**Review by hhyyrylainen — APPROVED** [review]
I don't fully see the need for this, but from a functionality / code standpoint I don't see anything wrong with this.

**Pull request merged**